Plate_2_data/4.processing_features/scripts/annotate_sc_plate2.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Annotate merged single cells with metadata from platemap file
+
+# ## Import libraries
+
+# In[1]:
+
+
+import pandas as pd
+import pathlib
+
+from pycytominer import annotate
+from pycytominer.cyto_utils import output
+
+import sys
+sys.path.append("../../utils")
+import sc_extraction_utils as sc_utils
```

Comment:
```suggestion
import sys
import pathlib

import pandas as pd
from pycytominer import annotate
from pycytominer.cyto_utils import output

sys.path.append("../../utils")
import sc_extraction_utils as sc_utils
```

Eric has been stressing following the PEP conventions for imports in my PRs and I think we as a lab should start focusing on this more. The [PEP documentation](https://peps.python.org/pep-0008/#imports) says imports should be in the following order:

1. Standard library imports.
2. Related third party imports.
3. Local application/library specific imports.

Reply:
Also, consider this comment for the rest of the python files in this PR and elsewhere!

Reply:
Huh, I haven't seen this! I agree that the lab should follow the same conventions so I will make this change for all notebooks/py files!

Reply:
See [45c5ceb](https://github.com/WayScience/pyroptosis_signature/pull/27/commits/45c5ceb3358d2427e9742a1a0701bbcadf78fc49) for change.